docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,240 @@
+tray icon and select 'Switch to Windows containers...' in order to run
+Docker images based on Windows OSs. This command takes a few seconds to
+execute.
+
+Moving your application involves these steps:
+
+1. Creating a publish task to build the assets for an image.
+2. Building a Docker image that will run your application.
+3. Starting a Docker container that runs your image.
+4. Verifying the application using your browser.
+
+## Publish script
+
+You can use the Visual Studio Publish command to create a publish profile
+for your application. You'll copy all the published assets to your Docker
+image later in this tutorial. The publish command will put all the publish
+assets in one directory tree that you will copy to your target image.
+Right-click on your solution in Visual Studio, and select "Publish".
+Click the "Custom" profile button, and then select "File System" as the
+method (see image). Choose a new directory in your solution folder. (The
+downloaded sample uses "containerImage" as the folder name).
+
+![Publish Connection][publish-connection]
+
+Next, open the "Advanced" section of the "File Publish" options. Select
+"Precompile during publishing". This optimization means that you be
+compiling views in the Docker container, you are copying the precompiled
+views.
+
+![Publish Settings][publish-settings]
+
+Click publish, and Visual Studio will copy all the needed assets to the
+destination folder. 
+
+## Build the image
+
+To run your application, you'll build an image based on the `microsft/aspnet`
+image located on [Docker Hub](https://hub.docker.com/r/microsoft/aspnet/).
+
+You'll compose this image using the `microsoft/aspnet` image as the base:
+
+```
+FROM microsoft/aspnet
+```
+
+The next command creates the directory that holds your application:
+
+```
+RUN mkdir C:\randomanswers
+```
+
+After that, you run a powershell command in the Docker image to setup
+the new site: 
+
+```
+RUN powershell -NoProfile -Command \
+    Import-module IISAdministration; \
+    New-IISSite -Name "ASPNET" -PhysicalPath C:\randomanswers -BindingInformation "*:8000:"
+```
+
+Following that, you instruct your Docker image to listen on port 8000: 
+
+```
+EXPOSE 8000
+```
+
+And last, you copy everything from your publish directory to the Docker image:
+
+```
+ADD containerImage/ /randomanswers
+```
+
+Run a Docker build command to create the image that will run your asp.net
+application. Open a powershell window, and type the following command in
+the solution directory:
+
+```
+docker build -t mvcrandomanswers .
+```
+
+Once that command completes, you can run the `docker images` command
+to see information on the new image:
+
+```
+REPOSITORY                    TAG                 IMAGE ID            CREATED             SIZE
+mvcrandomanswers              latest              86838648aab6        2 minutes ago       8.104 GB
+```
+
+The IMAGE ID will be different on your machine. Now, let's run the applicaton.
+
+## Start a container
```

Comment:
Additionally, as someone coming from Linux containers it can strike you as rather odd that there is no entrypoint.

I understand that this is a tutorial, but nowhere have we explained what the base image does for you.
You have to create the site, but don't have to start the server? WHY?